### PR TITLE
add neuroscience section

### PR DESCRIPTION
**Pull request by jdkent**

### CODE REVIEW

**Comment by vsoch** [comment]
Great idea! Would you mind putting the sections in alphabetical order (or giving me the A-ok to do so?) It also looks like line 75 is one space too close to the left margin.

**Comment by jdkent** [comment]
thanks for catching those, did I fix both in the most recent commit?

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ If you want to get involved with science and research software engineering, this
    - [Academic Labs](#academic-labs) 
    - [National Labs](#national-labs) 
  - [Psychology](#psychology)
+ - [Neuroscience](#neuroscience)
```
Comment (vsoch) [inline README.md:25]: Almost there! This link just needs to be moved up to be before Psychology.

Comment (jdkent) [reply README.md:25]: d'oh, should be fixed now

Comment (vsoch) [reply README.md:25]: Perfecto!

**Pull request merged**